README.md

```diff
@@ -103,6 +103,18 @@ to troubleshoot an issue in development.
 BetterErrors.editor = :mvim
 ```
 
+
+##Set maximum variable size for inspector.
+
+```ruby
+# e.g. in config/initializers/better_errors.rb
+# This will stop BetterErrors from trying to render larger objects, which can cause
+# slow loading times and browser performance problems.
+# default value: 100_000
+BetterErrors.maximum_variable_inspect_size = 100_000
```

Comment:
At the moment it's purely bytes, it's a very naive approach to get things moving. Happy to look at modifying this if we think there is a better approach, and I'll certainly add this caveat to documentation.